_includes/footer.html

```diff
@@ -1,74 +1,90 @@
-<div id="footer">
+<section class="footer">
 	<div class="container">
-		<div class="row footer_nav_row">
-			<div class="col-xs-12 col-md-9">
-				<div class="row">
-					<div class="footer_nav_col col-xs-6 col-sm-4 col-md-3">
-						<ul class="footer_nav">
-							<li class="header">Company</li>
-							<li><a href="https://www.cockroachlabs.com/blog/">Blog</a></li>
-							<li><a href="https://www.cockroachlabs.com/careers/">Careers</a></li>
-							<li><a href="https://www.cockroachlabs.com/community/events">Events</a></li>
-							<li><a href="https://www.cockroachlabs.com/press/">Press</a></li>
-							<li><a href="https://www.cockroachlabs.com/contact/">Contact Us</a></li>	
-						</ul>
-					</div>
-					<div class="footer_social_col col-xs-6 col-sm-4 col-md-3">
-						<ul class="footer_social">
-							<li class="header">Connect</li>
-							<li><a target="_blank" href="https://twitter.com/cockroachdb">Twitter</a></li>
-							<li><a target="_blank" href="https://gitter.im/cockroachdb/cockroach">Gitter</a></li>
-							<li><a target="_blank" href="https://github.com/cockroachdb/cockroach">GitHub</a></li>
-							<li><a href="https://groups.google.com/forum/#!forum/cockroach-db">Google Groups</a></li>
-							<li><a target="_blank" href="http://www.reddit.com/r/CockroachDB/">Reddit</a></li>
-						</ul>
-					</div>
-					<div class="footer_documentation_col col-xs-12 col-sm-4 col-md-3">
-						<ul class="footer_documentation">
-							<li class="header">Documentation</li>
-							<li><a href="{{ "/install-cockroachdb.html" | prepend:site.baseurl }}" data-proofer-ignore>Install</a></li>
-							<li><a href="{{ "/start-a-local-cluster.html" | prepend:site.baseurl }}" data-proofer-ignore>Quickstart</a></li>
-							<li><a href="{{ "/frequently-asked-questions.html" | prepend:site.baseurl }}" data-proofer-ignore>FAQ</a></li>
-							<li><a href="{{ "/improve-the-docs.html" | prepend:site.baseurl }}" data-proofer-ignore>Contribute</a></li>
-						</ul>
-					</div>
-					<div class="col-xs-6 col-sm-3">
-						<div class="contact_col">
-							<!--
-							<div class="contact_header">Contact</div>
-							<div class="address address_one">110 5th Avenue </div>
-							<div class="address address_two">NY, NY 10011</div>
-							<div class="contact_email"><a href="mailto:info@cockroachlabs.com" target="_blank">info@cockroachlabs.com</a></div>
-							-->
-						</div>
-					</div>
-				</div>
+		<div class="row">
+			<div class="company-nav-col col-xs-6 col-sm-4 col-md-15">
+				<ul class="footer-nav">
+					<li class="header">Company</li>
+					<li><a href="http://www.cockroachlabs.com/about/" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-depth-0">About</a></li>
```

Comment:
`http://www.cockroachlabs.com` -> `https://www.cockroachlabs.com`
There are a bunch more in this file and others in this PR.